src/Lasso.jl

```diff
@@ -301,10 +322,191 @@ function StatsBase.fit{T<:AbstractFloat,V<:FPVector}(::Type{LassoPath},
 
     # Fit path
     path = LassoPath{typeof(model),T}(model, nulldev, nullb0, λ, autoλ, Xnorm)
-    dofit && fit!(path; irls_tol=irls_tol, fitargs...)
+    if dofit
+        fit!(path; irls_tol=irls_tol, fitargs...)
+    else
+        path.λ = zeros(T, 0)
+        path.pct_dev = zeros(T, 0)
+        path.coefs = spzeros(T, p, 0)
+        path.b0 = zeros(T, 0)
+        path.niter = 0
+    end
     path
 end
 
+StatsBase.nobs(path::RegularizationPath) = length(path.m.rr.y)
+
+# TODO would be better to use GLM.dispersion_parameter, but that breaks backward comp.
+dispersion_parameter(path::RegularizationPath) = typeof(distfun(path)) in (Gamma, Normal, InverseGaussian)
+
+function StatsBase.loglikelihood(path::RegularizationPath)
+    n = nobs(path)
+    if typeof(path.m) <: LinearModel
+        -0.5*n.*log(deviance(path))
+    else
+        -0.5*n.*deviance(path)
+    end
+end
+
+if Pkg.installed("StatsBase") >= v"0.8.0"
+    import StatsBase.df, StatsBase.aicc
+end
+
+""" Approximates the degrees-of-freedom in each segment of the path as the number of non zero coefficients
```

Comment:
The Julia docstring convention is generally to write the function name indented on the first line and leave following lines unindented, e.g. https://github.com/JuliaStats/StatsBase.jl/blob/add47e6b0b44c26540e6bdabd4ce3470582d58ae/src/statmodels.jl